.vscode/scripts/import-example-app.sh

```diff
@@ -27,10 +29,18 @@ else
   rm -rf app/
   cp -a $VELOCITAS_EXAMPLES_PATH/$CHOSEN_EXAMPLE/. app/
 
+  # Re-compile requirements*.txt (including app and tests one)
   pip-compile -r -q ./requirements.in
-  pip-sync ./requirements.txt ./app/requirements.txt ./app/tests/requirements.txt
+  # Re-intstall necessary packages in DevContainer
+  for file in ./requirements.txt ./app/requirements.txt ./app/tests/requirements.txt
```

Comment:
please add requirements-links.txt even if it is empty

Reply:
added